SEOImages: User Story 1463880, Part 29 (#11409)
dotnet/docs: docs/visual-basic/programming-guide/language-features/control-flow/decision-structures.md

  
 The following illustration shows a decision structure that tests for a condition being true and takes different actions depending on whether it is true or false.  
  

>  ![Diagram that shows an If...Then...Else construction.](./media/decision-structures/if-then-else-construction.gif)  

Reviewer: ```suggestion
 ![A flow chart of an If...Then...Else construction.](./media/decision-structures/if-then-else-construction.gif)  
```
